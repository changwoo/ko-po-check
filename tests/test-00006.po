# -*-text-*-
# test 6 - 의존명사, etc.
#
msgid ""
msgstr ""
"Project-Id-Version: test 6\n"
"POT-Creation-Date: 2002-07-27 01:30+0900\n"
"PO-Revision-Date: 2002-07-27 01:30+0900\n"
"Last-Translator: Changwoo Ryu <cwryu@debian.org>\n"
"Language-Team: Korean <ko@li.org>\n"
"MIME-Version: 1.0\n"
"Content-Type: text/plain; charset=EUC-KR\n"
"Content-Transfer-Encoding: 8bit\n"

# E: '읽을 수'
#: test.c:6
msgid "Can't read file"
msgstr "파일을 읽을수 없습니다"

# E: '수 없습니다'
#: test.c:6
msgid "Can't find file"
msgstr "파일을 찾을 수없습니다"

# E: '열 수 없습니다'
#: test.c:6
msgid "Can't open file"
msgstr "파일을 열수없습니다"

# E: '열 수 없습니다'
#: test.c:6
#, c-format
msgid "unknown machine mode `%s'"
msgstr "알수 없는 머신 모드 `%s'"

# E: '열 수 없습니다'
#: test.c:6
#, c-format
msgid "unknown machine mode `%s'"
msgstr "알수 없는 머신 모드 `%s'"

# E: 'return할 때"
#: test.c:6
msgid "abort if a function returns"
msgstr "return할때 abort"
