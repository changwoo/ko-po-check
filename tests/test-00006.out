test-00006.po:18: "읽을수 ": 의존 명사는 띄어 써야 합니다
test-00006.po:23: "찾을 수없": 의존 명사는 띄어 써야 합니다
test-00006.po:28: "열수없": 의존 명사는 띄어 써야 합니다
test-00006.po:34: "알수 ": 의존 명사는 띄어 써야 합니다
test-00006.po:40: "알수 ": 의존 명사는 띄어 써야 합니다
test-00006.po:45: "할때 ": `~할 때'라고 띄어 써야 합니다
